RoddyUI/src/de/dkfz/roddy/client/fxuiclient/DataSetView.groovy

```diff
@@ -487,13 +487,8 @@ public class DataSetView extends CustomControlOnBorderPane implements Initializa
 //        loadFilesCreatedByWorkflow(ec);
 
         clearJobList();
-<<<<<<< HEAD:RoddyUI/src/de/dkfz/roddy/client/fxuiclient/DataSetView.groovy
```

Comment:
I actually do not know, whether I solved this correctly. Would you have a deep look here?

Reply:
This only pulls the JobInfoObject class from a specific namespace and substitutes the now non-existing FakeJob class by FakeBEJob. Seems reasonable.

What else should I check here, also given that the GUI is not going to be maintained anymore.